pkgs/by-name/pa/payme/package.nix

```diff
@@ -8,11 +8,27 @@ buildGoModule rec {
     owner = "jovandeginste";
     repo = "payme";
     rev = "v${version}";
-    hash = "sha256-WE/sAs0VSeb5UKkUy1iyjyXtgDmlQhdZkw8HMMSbQiE=";
+    hash = "sha256-2gZgmYgLaJQRQ+3VOUDnMm5QBjfKyxyutVf9NrbGO3g=";
```

Comment:
This is somewhat dangerous, since it modified the hash.

However, given that I can't provide a better solution right now, I will not pursue this.

Reply:
The easiest solution would be to just fill that has with zeros and don't worry about it but if you want to go the complicated route, sure.